content/en/docs/reference/resource-types/_index.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Resource Types
+---
+
+<!-- overview -->
+
+This section provides information on the different resource types you can request, built into Kubernetes.
+
+<!-- body -->
+
+There are 3 different resource types: `cpu`, `memory`, and `storage`.
```

Comment:
Hi. There is also `hugepages-*` - see https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#resource-requests-and-limits-of-pod-and-container

and, as a beta feature, `ephemeral-storage` - see https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#local-ephemeral-storage and https://kubernetes.io/docs/concepts/policy/resource-quotas/#storage-resource-quota

Reply:
Copy, I'll add these!